tests/component/records/pandas/test_to_csv_options.py

```diff
@@ -3,79 +3,21 @@
 import io
 from typing import Set
 from ..datetime_cases import (
-    DATE_CASES, DATETIMETZ_CASES, DATETIME_CASES, TIMEONLY_CASES, create_sample,
+    DATETIMETZ_CASES, DATETIME_CASES, create_sample,
     SAMPLE_YEAR, SAMPLE_MONTH, SAMPLE_DAY, SAMPLE_HOUR, SAMPLE_MINUTE, SAMPLE_SECOND
 )
 from records_mover.records.pandas.to_csv_options import pandas_to_csv_options
 from records_mover.records import DelimitedRecordsFormat, ProcessingInstructions
 
 
 class TestToCsvOptions(unittest.TestCase):
```

Comment:
The rest of this is generally removing special cases from the tests. Let me know if you have questions about any of it!

Reply:
Am I right in understanding the change as follows:

* Pandas exports all date/time objects as date+time strings using the single datetime format field
* Because of that, we always need the date/time/datetime format hints to be consistent for pandas.
* The datetime test cases cover all the interesting combos of date and time formats
* Now, with this PR, we handle basically all kinds of date/time formats for pandas export.
* Therefore, there is no reason to independently test date or time formatting, since they aren't independent knobs.

Is that right?

Reply:
You got it!